scripts/setup_build_env.sh

```diff
@@ -124,14 +123,17 @@ function install_ubuntu_package_dependencies()
         gcc-multilib \
         systemtap-sdt-dev \
         librdmacm-dev \
+        libibverbs-dev \
         libfdt-dev \
         autoconf \
         automake \
         autotools-dev \
         libtool \
         grpc-proto \
         protobuf-compiler-grpc \
-        libgrpc10
+        libgrpc10 \
+        dracut \
+        linux-headers-"$(uname -r)"
```

Comment:
Ok, maybe this one could be improved as you do not collect return code from shell execution (probably do not need to most of the time) and this triggers shell-check warnings. Can we have it done like:
```bash
{
local APT_LINUX_HEADERS="linux-headers-$(uname -r)"
# Install package dependencies
apt-get update --fix-missing && \
apt-get install --no-install-recommends -y \
       ...
       "${APT_LINUX_HEADERS}"
}
```